common/pkgfiles/referencedfilelist.cpp

```diff
@@ -533,7 +533,15 @@ IPropertyTree *ReferencedFile::getRemoteStorageFileTree(IUserDescriptor *user, c
 
     Owned<wsdfs::IDFSFile> dfsFile = wsdfs::lookupDFSFile(remoteLFN.str(), AccessMode::readSequential, INFINITE, wsdfs::keepAliveExpiryFrequency, user);
     IPropertyTree *tree = (dfsFile) ? dfsFile->queryFileMeta() : nullptr;
-    return tree ? tree->getPropTree("File") : nullptr;
+    if (!tree)
+    {
+        DBGLOG("RemoteStorage FileMetaTree not found %s [remoteStorage=%s, prefix=%s]", remoteLFN.str(), nullText(remoteStorageName), nullText(remotePrefix));
+        return nullptr;
+    }
+    tree = tree->getPropTree("File");
```

Comment:
very minor comment:  I would introduce a new variable IPropertyTree * fileTree, because it is potentially confusing that the first time it is assigned to it is not linked, but the second time it is.